TextFormat.md

```diff
@@ -75,11 +92,40 @@ official format may look entirely different:
   [wabt]: https://github.com/WebAssembly/wabt
   [binaryen]: https://github.com/WebAssembly/binaryen
 
-# Debug symbol integration
+## Debug symbol integration
 
 The binary format inherently strips names from functions, locals, globals, etc,
 reducing each of these to dense indices. Without help, the text format must
 therefore synthesize new names. However, as part of the [tooling](Tooling.md)
 story, a lightweight, optional "debug symbol" global section may be defined
 which associates names with each indexed entity and, when present, these names
 will be used in the text format projected from a binary WebAssembly module.
+
+# Future design :unicorn:
```

Comment:
Elsewhere it had been suggested we strive to make all unicorns links.
On the other hand this doesn't really have anywhere else to link to.